lit_tests/kernel/wave/attention.py

```diff
@@ -1087,10 +1087,20 @@ def repeat(
 
 @run_test
 def test_paged_flash_decoding():
-    # (B, M, N, K1, K2, BH, S)
-    shape = (128, 1, 32, 32, 64, 4, 8, 128)
-    max_tokens = 2048
+    B = 128
+    M = 1
```

Comment:
Is `M` used anywhere?

Reply:
No, this is no longer required. Will remove.